Integrations/SecBI/SecBI.py

```diff
@@ -0,0 +1,203 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import requests
+from typing import Dict, Tuple
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+SECBI_API_ROOT = '/api/v1'
+SECBI_API_ENDPOINT_STATUS = SECBI_API_ROOT + '/status'
+SECBI_API_ENDPOINT_INCIDENT = SECBI_API_ROOT + '/thirdparties/integration/incident'
+SECBI_API_ENDPOINT_INCIDENTS = SECBI_API_ROOT + '/thirdparties/integration/incidents'
+SECBI_INCIDENT_FIELDS = ['id', 'host', 'identity', 'internalIp', 'sIp', 'firstAppearance', 'lastAppearance']
+
+''' HELPER FUNCTIONS '''
+
+
+def capitalize(s: str):
```

Comment:
Use Python [builtin capitalize function](https://docs.python.org/3/library/stdtypes.html#str.capitalize) instead

Reply:
The built-in capitalize will lowercase the rest of the string. I only want to capitalize the first character and leave the rest as is. For example, I want `firstAppearance` to become `FirstAppearance` and not `Firstappearance`